packages/fx-core/src/component/middleware/envMW.ts

```diff
@@ -37,7 +37,9 @@ export const envLoaderMWImpl: Middleware = async (ctx: CoreHookContext, next: Ne
       ctx.result = err(envListRes.error);
       return;
     }
-    question.staticOptions = envListRes.value;
+    question.staticOptions = envListRes.value.filter(
```

Comment:
What's this change for?

Reply:
Is it possible to ensure `inputs.env` is always set for F5, then no need to have this change?

Reply:
For now, core support multi env. So, we need to exclude 'local' environment.
![image](https://user-images.githubusercontent.com/13211513/201034975-c2efa187-e0a9-4c8c-b6c2-99fb3e0f5b3d.png)

Reply:
So this is not for F5?
I prefer not adding local-specific logic. `local` is just the default name and user can customize any env name (`local1`, `my-desktop`, `my-laptop`, ...)
And, what if user really want to provision `local` environment?

Reply:
BTW, for F5 (prepare teams app deps), let's just hard code `local` as the default env for the 1st release.
I believe in later release we need some config/setting to let user to choose which env to be prepared.

Reply:
In addition, if there should be local-specific logic, add to the very top level (extension, template, user settings) instead of core.

Reply:
Make sense, I will change it back.